src/theory/bv/theory_bv_rewrite_rules.h

```diff
@@ -621,81 +619,92 @@ struct AllRewriteRules {
   RewriteRule<UltOnes> rule147;
 };
 
-template<> inline
-bool RewriteRule<EmptyRule>::applies(TNode node) {
+template <>
+inline bool RewriteRule<EmptyRule>::applies(TNode node)
+{
   return false;
 }
 
-template<> inline
-Node RewriteRule<EmptyRule>::apply(TNode node) {
-  Trace("bv-rewrite") << "RewriteRule<EmptyRule> for " << node.getKind() <<"\n"; 
+template <>
+inline Node RewriteRule<EmptyRule>::apply(TNode node)
+{
+  Trace("bv-rewrite") << "RewriteRule<EmptyRule> for " << node.getKind()
+                      << "\n";
   Unreachable();
   return node;
 }
 
-template<Kind kind, RewriteRuleId rule>
-struct ApplyRuleToChildren {
-
-  static Node apply(TNode node) {
-    if (node.getKind() != kind) {
+template <Kind kind, RewriteRuleId rule>
```

Comment:
Ditto, can you exclude white space changes for parts of this file you didn't modify?

Reply:
reverted